### ISSUE TITLE
Placeholder texts in form field too light

### ISSUE THREAD

**Issue by jaakkocsc** [CSCfi/rems#2229]
More stronger colour would be more readable. Should follow accessibility requirements.

- [ ] check that contrast for THL theme placeholder text is enough (maybe should be stronger than now anyway?)
